Samples~/AvatarCreatorSamples/AvatarCreatorWizard/Scripts/UI/SelectionScreens/AvatarCreatorSelection.cs

```diff
@@ -198,10 +201,32 @@ private async Task LoadAvatarColors()
         {
             var startTime = Time.time;
             var colors = await avatarManager.LoadAvatarColors();
-            assetButtonCreator.CreateColorUI(colors, UpdateAvatar);
+            var equippedColors = GetEquippedColors();
+
+            assetButtonCreator.CreateColorUI(colors, UpdateAvatar, equippedColors);
             SDKLogger.Log(TAG, $"All colors loaded in {Time.time - startTime:F2}s");
         }
 
+        private Dictionary<AssetType, int> GetEquippedColors()
+        {
+            var colorAssetTypes = AssetFilterHelper.GetAssetTypesByFilter(AssetFilter.Color).ToHashSet();
```

Comment:
```suggestion
            var colorAssetTypes = AssetTypeHelper.GetAssetTypesByFilter(AssetFilter.Color).ToHashSet();
```

Small typo here, I think you might have renamed the class and just forgotten to update this one.